### PR TITLE
Reload diagnostics when changing vscode/local config

### PR DESCRIPTION
**Pull request by nobodywasishere**
Part 1.5, review / merge #147 first

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -99,6 +105,17 @@ export function activate(context: ExtensionContext) {
...
 
+function executeAmebaOnWorkspace(ameba: Ameba | null) {
+    if (!ameba) return;
+
+    workspace.textDocuments.forEach(doc => {
+        if (isValidCrystalDocument(doc)) {
+            outputChannel.appendLine(`[Init] Running ameba on ${getRelativePath(doc)}`);
+            ameba.execute(doc);
+        }
+    });
```
Comment (Sija) [inline src/extension.ts:116]:
```suggestion
    for (const doc of workspace.textDocuments) {
        if (isValidCrystalDocument(doc)) {
            outputChannel.appendLine(`[Init] Running ameba on ${getRelativePath(doc)}`);
            ameba.execute(doc);
        }
    }
```

File: src/extension.ts [hunk 2]
```diff
@@ -99,6 +105,17 @@ export function activate(context: ExtensionContext) {
 
 export function deactivate() { }
 
+function executeAmebaOnWorkspace(ameba: Ameba | null) {
+    if (!ameba) return;
+
+    workspace.textDocuments.forEach(doc => {
+        if (isValidCrystalDocument(doc)) {
+            outputChannel.appendLine(`[Init] Running ameba on ${getRelativePath(doc)}`);
```
Comment (Sija) [inline src/extension.ts:113]: I'd change the `Init` prefix to sth more apt.

```suggestion
            outputChannel.appendLine(`[Init] Running ameba on ${getRelativePath(doc)}`);
```

**Review by Sija — APPROVED** [review]
LGTM

**Review by veelenga — APPROVED** [review]
lgtm

**Pull request merged**